index.js

```diff
@@ -1,39 +1,267 @@
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
+  async function stateTwo() {
+    //TO DO: THIS WILL BE OUR SECOND STATE, IN THE FOYER WITH SEVEN DAYS
+    let secondLoop = true;
+    let answer2 = await ask(
+      "Welcome to the foyer at Burlington Code Academy. In front of you is a set of stairs going up. In the corner is a copy of Seven Days. \n"
+    );
+    while (secondLoop === true) {
+      if (answer2.includes("back") || answer2.includes("leave")) {
+        begin();
+        secondLoop = false;
+      } else if (answer2.includes("take")) {
+        answer2 = await ask("You have taken a copy of Seven Days \n");
+        inv.newsPaper = "Seven Days";
+      } else if (answer2.includes("stairs" || answer2.includes("upstairs"))) {
+        stateThree();
+        secondLoop = false;
+      } else if (answer2.includes("inv") || answer2 === "i") {
+        showInv();
+        answer2 = await ask("\n");
+      } else if (answer2.includes("read")) {
+        console.log(
+          "Seven Days features an article about how Johnson State's Rugby team was decimated by a group of 5th graders. \n"
+        );
+        answer2 = await ask("");
+      } else {
+        answer2 = await ask("I'm sorry, I cannot" + answer2 + "\n");
+      }
+    }
+  }
+  /*
+SECOND ROOM ENDS/BEGIN THIRD ROOM ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
+  async function stateThree() {
+    //TO DO: THIS WILL BE THE THIRD FLOOR HALLWAY
+    let thirdLoop = true;
+    let answer3 = await ask(
+      "You are in the third floor hallway. To your right is a bathroom, down the hall is Eternity Web and the Burlington Coding Academy \n"
+    );
+    while (thirdLoop === true) {
+      if (answer3.includes("eternity")) {
+        stateSix();
+        thirdLoop = false;
+      } else if (
+        answer3.includes("classroom") ||
+        answer3.includes("academy") ||
+        answer3.includes("coding")
+      ) {
+        stateFive();
+        thirdLoop = false;
+      } else if (answer3.includes("bathroom")) {
+        stateFour();
+        thirdLoop = false;
+      } else if (answer3.includes("inv") || answer3 === "i") {
+        showInv();
+        answer3 = await ask("\n");
+      } else {
+        answer3 = await ask("Im sorry, I cannot " + answer3 + "\n");
+      }
+    }
+  }
+  /*
+THIRD ROOM ENDS/FOURTH ROOM BEGINS ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
+  async function stateFour() {
+    //THIS WILL BE THE BATHROOM
+    let fourthLoop = true;
+    let answer4 = await ask(
+      "Welcome to the bathroom. There's nothing interesting here. Perhaps you should go back to the hall. \n"
+    );
+    while (fourthLoop === true) {
+      if (answer4.includes("hall")) {
```

Comment:
For many of the rooms the transition commands are not very intuitive. Consider adding more edge cases, or using a generic function to handle state changes.